sound/soc/sof/intel/hda.h

```diff
@@ -158,6 +158,12 @@
 #define HDA_DSP_MBOX_UPLINK_OFFSET		0x81000
 
 #define HDA_DSP_STREAM_RESET_TIMEOUT		300
+/*
+ * Timeout in us, for setting the stream RUN bit, during
+ * start/stop the stream. The timeout expires if new RUN bit
+ * value cannot be read back within the specified time.
+ */
+#define HDA_DSP_STREAM_RUN_TIMEOUT		300
```

Comment:
still doesn't tell me where this value comes from, is this a hardware recommendation or 'same as the other drivers'

Reply:
@plbossart it is "same as the other drivers"